core/electron/src/backend/ElectronHost.ts

```diff
@@ -259,7 +259,14 @@ export class ElectronHost {
       this._ipc = new ElectronIpc();
       const app = this.app;
       if (!app.isReady())
-        this.electron.protocol.registerSchemesAsPrivileged([{ scheme: "electron", privileges: { standard: true, secure: true } }]);
+        this.electron.protocol.registerSchemesAsPrivileged([{
+          scheme: "electron",
+          privileges: {
+            standard: true,
+            secure: true,
+            supportFetchAPI: true,

```

Comment:
Before we merge this are there any security implications to doing this?  @ben-polinsky @aruniverse @evelynpreslar-bentley

Reply:
My understanding is that this is enabling clients to use fetch to access electron://* urls. If that's the case, I'm not yet familiar with what that exposes.

Reply:
Ben and I were looking at the code and think that so long as fetch requests still go through the electron file protocol that we register, then the limitations and security on the filepath that the protocol plus the `parseElectronUrl` function provides shouldn't be any different.  

```ts
// handle any "electron://" requests and redirect them to "file://" URLs
this.electron.protocol.registerFileProtocol("electron", (request, callback) =>
                                             callback(this.parseElectronUrl(request.url)));
```

I checked, and the fetch requests for localization do indeed still go through `parseElectronUrl`.